src/Rector/Contrib/Nette/DI/ExpandFunctionToParametersArrayRector.php

```diff
@@ -73,7 +74,7 @@ public function isCandidate(Node $node): bool
      */
     public function refactor(Node $methodCallNode): ?Node
     {
-        /** @var Node\Scalar\String_ $argument */
+        /** @var String_ $argument */
```

Comment:
:+1: 

This could be covered by fixer in Symplify\CodingStandard: https://github.com/Symplify/Symplify/tree/master/packages/CodingStandard#types-should-not-be-referenced-via-a-fullypartially-qualified-name-but-via-a-use-statement-wrench

But it now lacks support for doc types. Would you be able to add so this is CI checked? That'd be great!

Reply:
Gonna add it in another PR to keep one feature per :smile: